rules-topics/19_status-effects.md

```diff
@@ -46,84 +46,95 @@ Shattering or destroying containers that contain things does NOT affect the item
 Skilless refers to any abilities purchased with Skill Points.  You do not lose any Armor or Health Points gained from purchased skills.
 
 
-## Status Effects
+## Ability
 
 ---
 | Term| Description|Encounter Call  |Visible?|
 |:---|---|:-----------------------------------------------:|:---:|
 | ACID SPITTLE| Ability user's ranged attacks deal x Acid Damage.  |  X Acid| No |
```

Comment:
We can remove this one.  Acid isn't a damage type we have anymore.

Reply:
easily done